tests/cases/compiler/contextualSignatureWithExtraParameters.ts

```diff
@@ -0,0 +1,55 @@
+// https://github.com/microsoft/TypeScript/issues/59309

```

Comment:
tip: I almost always include `// @strict: true` at the top of the test file unless I want to test non-strict behavior specifically. I already lost too many hours hunting down mismatched actual results from test files just to learn that the test was OK, it just wasn't executed with `strictNullChecks` or another strict option ;p

SImilarly, I usually use `@noEmit: true` to avoid `.js` baselines being generated redundantly if I only want to test type-checking behavior and not js/dts emit.